test/hotspot/jtreg/testlibrary/ctw/src/sun/hotspot/tools/ctw/CtwRunner.java

```diff
@@ -299,6 +299,7 @@ private String[] cmd(long classStart, long classStop) {
                 "-XX:+StressIGVN",
                 "-XX:+StressCCP",
                 "-XX:+StressMacroExpansion",
+                "-XX:+StressIncrementalInlining",
```

Comment:
I think we want to maintain the historical order here. Check me on this: `StressIncrementalInlining` went in before `StressMacroExpansion`, but after `StressCCP`, and so should be one line up?